catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/MetacardFactory.java

```diff
@@ -56,50 +59,38 @@ Metacard generateMetacard(String mimeTypeRaw, String id, String fileName, Path t
             throws MetacardCreationException, MimeTypeParseException {
 
         Metacard generatedMetacard = null;
-        InputTransformer transformer = null;
-        StringBuilder causeMessage = new StringBuilder("Could not create metacard with mimeType ");
-        try {
-            MimeType mimeType = new MimeType(mimeTypeRaw);
-
-            List<InputTransformer> listOfCandidates = mimeTypeToTransformerMapper.findMatches(
-                    InputTransformer.class,
-                    mimeType);
-
-            LOGGER.debug("List of matches for mimeType [{}]: {}", mimeType, listOfCandidates);
-
-            for (InputTransformer candidate : listOfCandidates) {
-                transformer = candidate;
-
-                try (InputStream transformerStream = com.google.common.io.Files.asByteSource(
-                        tmpContentPath.toFile())
-                        .openStream()) {
-                    generatedMetacard = transformer.transform(transformerStream);
-                }
-                if (generatedMetacard != null) {
-                    break;
-                }
+
+        MimeType mimeType = new MimeType(mimeTypeRaw);
+
+        List<InputTransformer> listOfCandidates = mimeTypeToTransformerMapper.findMatches(
+                InputTransformer.class,
+                mimeType);
+        List<String> stackTraceList = new ArrayList<>();
+
+        LOGGER.debug("List of matches for mimeType [{}]: {}", mimeType, listOfCandidates);
+
+        for (InputTransformer candidate : listOfCandidates) {
```

Comment:
This seems to make the logic in https://github.com/codice/ddf/blob/5113a905dab89696fe97f18418424397d5e50ace/catalog/rest/catalog-rest-endpoint/src/main/java/org/codice/ddf/endpoints/rest/RESTEndpoint.java#L1161-L1161 redundant, or at the very least it's code duplication. Can we remove the logic from the `RESTEndpoint.generateMetacard`, or dedupe the code if that's not feasible?